velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -697,35 +697,4 @@ struct ArrayUnionFunction {
   }
 };
 
-template <typename T>
-struct ArrayUnionFunctionString {
```

Comment:
Why removing this?

Reply:
zero-copy string compute is not safe for more than one args. Since zero-copy is not used, no point to create a separate function for it.

Reply:
https://github.com/facebookincubator/velox/issues/5779

Reply:
Can you comment this in PR description? Thanks!